src/main/scala/cromwell/engine/WorkflowDescriptor.scala

```diff
@@ -99,7 +98,7 @@ object WorkflowDescriptor {
   }
 
   def apply(id: WorkflowId, sourceFiles: WorkflowSourceFiles, conf: Config): WorkflowDescriptor = {
-    validateWorkflowDescriptor(id, sourceFiles, CromwellServer.backend, conf) match {
```

Comment:
That's not *completely* true. The object's construction of WMA references it as well. This all stems from a change I'm not particularly proud of but was the lesser of two hopefully temporary evils a while back